### ISSUE TITLE
Fix css of the pages. 

### ISSUE THREAD

**Issue by AryanGupta0007** [TeamCodeArena/DeliverEase#64]
There are some pages in the project whose design is in bad shape so we need to fix the css of those pages.

**Comment by SID262000** [comment]
Great work, @AryanGupta0007 👏

We might eventually transition into SCSS for extended flexibility over CSS.

Reference: https://www.google.com/amp/s/www.geeksforgeeks.org/what-is-the-difference-between-css-and-scss/amp/

FYI: Make sure to link the Issue number while committing your changes.

**Comment by SID262000** [comment]
Here's a great starting point to apply for the TDD approach for the CSS code. This will make our software more robust.

[Reference](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://levelup.gitconnected.com/how-to-unit-test-css-a4595706b26d&ved=2ahUKEwjy6Jv2mdSAAxVn1zgGHRiEDDc4ChCOOHoECAUQAQ&usg=AOvVaw3sb75mPPbVIaFwGP-iI0at)

You can additionally set [Chromatic](https://www.chromatic.com/docs/) for comparing the styles before and after modification.

cc: @AryanGupta0007

**Comment by Devansh-Singh-Parmar** [comment]
We can change the frontend to React, this way the code will be much reusable and easy to read plus more good looks
cc :- @SID262000 @AryanGupta0007

**Comment by SID262000** [comment]
I wouldn't recommend switching to a new framework just because of its look and feel alongside making it easier to use, because everything is built on top of vanilla JS, the more we stick to the foundational building block, the more robust the software would be because ultimately the customer doesn't care about the background details.

**Comment by Devansh-Singh-Parmar** [comment]
Okay then, let's continue with this, but there will be a lot of hard work for responsiveness

**Comment by SID262000** [comment]
We have to try to achieve that to make it robust at a large scale.

**Comment by Atrika10** [comment]
No worries about responsiveness @Devansh-Singh-Parmar,  will handle it.

**Comment by AryanGupta0007** [comment]
I think this issue is close worth @SID262000

**Comment by SID262000** [comment]
@AryanGupta0007 I don't think we have tested the tools attached as part of the reference. Please look into it and update here accordingly, Thanks

**Comment by AryanGupta0007** [comment]
@SID262000 i did not understand what is actually meant to be done. Actually I created this issue specifically to fix some css issues which are now fixed so i suggested to close this.

**Comment by SID262000** [comment]
We can take this issue during our weekly staff meeting scheduled at 10pm on Saturday and brainstorm the further action items for the same.
cc: @AryanGupta0007

**Comment by Devansh-Singh-Parmar** [comment]
I can fix some issues in css!

**Comment by Atrika10** [comment]
It would be helpful if we start working after adding html & css linter in the main branch.

Thanks

**Comment by aswinsaravna** [comment]
can i be assigned to this issue if possible? I have talked with devansh and he said i can join in

**Comment by Devansh-Singh-Parmar** [comment]
@aswinsaravna sure.

**Comment by Devansh-Singh-Parmar** [comment]
Not getting enough time to do it, it's taking a lot of time, I'll do it after a week then, unassigning it from myself to avoid confusion

**Comment by N3v1** [comment]
Hi everyone, do you need any help?

**Comment by aswinsaravna** [comment]
> Hi everyone, do you need any help?

Hi, let me know how i can contact you, are you on slack?

**Comment by N3v1** [comment]
> > Hi everyone, do you need any help?
> 
> Hi, let me know how i can contact you, are you on slack?

Yes you can dm me on Slack

Note: I'm unavailable for Huddles @aswinsaravna

**Comment by SID262000** [comment]
Any estimated completion time, @aswinsaravna 
cc: @N3v1

**Comment by SID262000** [comment]
cc: @Atrika10 @neelansh27 For assistance, Thanks!

**Comment by N3v1** [comment]
> Any estimated completion time, @aswinsaravna cc: @N3v1

Hi @SID262000, @aswinsaravna said on Slack that he needs help because he needs to analyze the code written with Python to fix the CSS issue. But unfortunately, he has no Python experience whatsoever. He needs help to imagine something in the frontend. But I can't help either as I don't have any Python experience either

cc: @aswinsaravna @Atrika10

**Comment by SID262000** [comment]
Thanks for the update & follow up, @N3v1 
I've updated @aswinsaravna in today's weekly meeting regarding the next action item.

**Comment by aswinsaravna** [comment]
Currently in the process of getting to know python and django, and have seen the frontend of the website, the main index, the font and everything was irregular, would require some more time, to finish, since today EOD might not be enough, feels like i am getting the hang of this, thanks all for being supportive, will provide a positive result by this week, hopefully in 3 days if things work out sooner.
Trying my best.

**Comment by SID262000** [comment]
It's approximately about 2 weeks since your last comment. 
Can you please update the latest progress, Thanks
cc: @aswinsaravna

**Comment by aswinsaravna** [comment]
for stylesheet.css and style.css
=================================

Improvements made:

- I have grouped the similar styles together (e.g., fonts, global styles, buttons).
- Added comments to explain the sections of the CSS.
- Removed redundant styles and consolidated common properties.
- Improved the indentation and formatting for better readability.
- Made the code more consistent in terms of naming and structure.

Basically i have made the css more optimized, but there is still loads of work to do in terms of achieving a more pleasing frontend design, learning that as well.


for buyerorseller.css
======================

Improvements made:

-Enclosed the font name 'Inter' in single quotes to ensure it's recognized as a font name.
-Changed the font size to use a relative unit (rem) instead of an absolute unit (px) for better scalability.
-Used font-weight property to specify boldness instead of font: bold; for better readability and maintainability.
-These changes make your CSS more standard and easier to manage.

for indexcontainer.css
======================
Improvements made:

-Consolidated common properties like color, font-family, font-size, and font-weight into the appropriate rule sets for better readability and maintainability.
-Removed unnecessary redundant styles.
-Applied consistent formatting and spacing throughout the code.
-Organized media queries from largest to smallest screen sizes for better readability.

for jobform.css
===============

Improvements made:

-Enclosed font names in single quotes.
-Used relative units (rem) for font size and responsive dimensions.
-Grouped similar media queries together for better organization.
-Improved formatting and spacing for better readability.
